.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: node_js
 node_js:
-  - "5"
+  - "6"
```

Comment:
We wouldn't want to remove 5 here.  Instead we just want to add 6.  This means that tests will need to pass with both versions.